client/src/App.js

```diff
@@ -56,9 +56,10 @@ const App = () => {
         <Route path="/PICResult7" element={<PICResult7 />} />
         <Route path="/YellowBinResult" element={<YellowBinResult />} />
         <Route path="/RedBinResult" element={<RedBinResult />} />
+        <Route path="/ScanImage" element={<ScanImage />} />
 
         <Route path="/Account" element={<PrivateRoute><Account /></PrivateRoute>}/>
-        <Route path="/ScanImage" element={<PrivateRoute><ScanImage /></PrivateRoute>}/>
+        {/* <Route path="/ScanImage" element={<PrivateRoute><ScanImage /></PrivateRoute>}/> */}
```

Comment:
not sure if you wanna merge this to main

Reply:
How do i not do that